Project1-WeatherRadar/user-Api/src/main/java/com/indiana/userApi/controller/WeatherController.java

```diff
@@ -57,9 +60,17 @@ public UserInfo getBookDetails(@PathVariable("id") Integer id){
 
 
 
-			UserInfo newUserInfo = new UserInfo(0,sessionRequestInfo.getEmailAddress()) ;
+			UserInfo newUserInfo = new UserInfo();
+			newUserInfo.setUserEmail(sessionRequestInfo.getEmailAddress());
 			newUserInfo = userInfoRespository.save(newUserInfo);
 
+			SessionInfo sessionInfo = new SessionInfo();
+			sessionInfo.setSessionTime(new Timestamp(System.currentTimeMillis()));
+			sessionInfo.setDateTime(Calendar.getInstance());
+			sessionInfo.setRadStation(sessionRequestInfo.getRadStation());
+			sessionInfo.setUserId(newUserInfo.getId());
+			sessionInfo = sessionInfoRepository.save(sessionInfo);
+
 
 //			SessionInfo sessionInfo = new SessionInfo(0,0, sessionRequestInfo.getEmailAddress(), null,new Calendar()) ;
```

Comment:
Do remove commented code. Merge this now for integration test. Raise another PR with this changes.